File: two_d_array/Assigement.c/Q_1.c
#include<stdio.h>
int main(){
    int r,c;
    printf ("enter row :");
    scanf("%d",&r);
    printf("Enter colume :");
    scanf("%d",&c);
    int arr[r][c];
    printf("Enter element :");
    for(int i=0;i<r;i++){
        for(int j=0;j<c;j++){
            scanf("%d",&arr[i][j]);
        }
    }
     for(int i=0;i<r;i++){
        for(int j=0;j<c;j++){
            printf("%d",arr[i][j]);
        }
        printf("\n");
    }
return 0;

}

File: Array_2D/Q_1.java
package Array_2D;

import java.util.Scanner;

public class Q_1 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);


        int row=sc.nextInt();
        int colume=sc.nextInt();

        int [][]arr=new int [row][colume];
        for(int i=0;i<row;i++){
            for(int j=0;j<colume;j++){
                arr[i][j]=sc.nextInt();
            }

        }
        for(int i=0;i<row;i++){
            for(int j=0;j<colume;j++){
                System.out.print(arr[i][j]);
            }
            System.out.println();
        }


    }
}
